### ISSUE TITLE
Unable to reproduce MoCha with librispeech

### ISSUE THREAD

**Issue by Jungwon-Chang** [hirofumi0810/neural_sp#238]
Happy new year Hiro!  I have another question about MoCha in ./examples/librispeech

in your paper "CTC-synchronous Training for Monotonic Attention Model(2020)", you've mentioned that LSTM-MoCha achieved  
WER of 5.3, 14.5 for librispeech's test-clean and test-other.

However, after I trained the model, I got an absurd WER, which was more than 100%
```python
WER (dev_other_960_wpbpe10000, ep:45): 143.08 %
```

So I looked at the decoding results, and there were lots of hypotheses that were trapped in a loop of several phrases. For example:
```
would not that then i ventured to ask impatiently make nieces as much of an idol as apollo at i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i'm as i (116_288045-116-288045-0031)

but there is more of moral tonic and the opening candid discussion of a subject like one hand than in the multitude platitudes the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the more the (116_288046-116-288046-0001)
```

the config for the model was
`conf/asr/mocha/lstm_mocha.yaml`
To give you more detail, I did not used any data augmentation, nor language model.
I'm dying to know whether I missed some steps.

**Comment by hirofumi0810** [comment]
@Jungwon-Chang I notice this. This happens only when using unidirectional LSTM on Librispeech (other corpora is OK). I'll check it again.

**Comment by Jungwon-Chang** [comment]
Thanks, maybe I'll have to test with other datasets

**Comment by Jungwon-Chang** [comment]
Thanks, maybe I'll have to test with other corpora

**Comment by hirofumi0810** [comment]
Hi @Jungwon-Chang, this issue was fixed via #265.

**Comment by Jungwon-Chang** [comment]
Thanks! I will try with the new configuration and let you know the results.
